format/index.md

```diff
@@ -873,15 +880,22 @@ for each relationship name can be specified:
 GET /articles/1?include=comments.author
 ```
 
-> Note: A request for `comments.author` should not automatically also
-include `comments` in the response. This can happen if the client already
-has the `comments` locally, and now wants to fetch the associated authors
-without fetching the comments again.
+> Note: Because compound documents require full linkage, intermediate
+resources in a multi-part path must be returned along with the leaf nodes.
+For example, a response to a request for `comments.author` should
+automatically include `comments` as well as the `author` of each of those
+`comments`.
+
+> Note: A server may choose to expose a deeply nested relationship such as
```

Comment:
I have changed that note to read:

> Note: A server may choose to expose a deeply nested relationship such as
`comments.author` as a direct relationship with an alias such as
`comment-authors`. This would allow a client to request
`/articles/1?include=comment-authors` instead of
`/articles/1?include=comments.author`. By abstracting the nested
relationship with an alias, the server can still provide full linkage in
compound documents without including potentially unwanted intermediate
resources.